app/static/js/events/wizard/vue-app.js

```diff
@@ -350,10 +357,22 @@ function shouldDisableMove($this) {
         $this.event.start_time_time.trim() === '' ||
         $this.event.start_time_date.trim() === '' ||
         $this.event.end_time_time.trim() ==='' ||
-        $this.event.end_time_date.trim() === ''
+        $this.event.end_time_date.trim() === '' ||
+        checkTickets($this.event.tickets)
     );
 }
 
+function checkTickets($this) {
+    var flag = false;
+    $this.forEach(function(ticket){
+        if(ticket.type === 'paid' && (ticket.price === '' || ticket.price <=0))
+        {
+            flag = true;
```

Comment:
Okay ... this is fine ... I had accidentally read this as `return true` while reviewing .. Sorry about that :)